tests/PHPStan/Analyser/data/bitwise-not.php

```diff
@@ -7,11 +7,13 @@
 /**
  * @param string|int $stringOrInt
  * @param non-empty-string $nonEmptyString
+ * @param non-falsy-string $nonFalsyString
  */
-function foo(int $int, string $string, float $float, $stringOrInt, string $nonEmptyString) : void{
+function foo(int $int, string $string, float $float, $stringOrInt, string $nonEmptyString, $nonFalsyString) : void{
 	assertType('int', ~$int);
 	assertType('string', ~$string);
 	assertType('non-empty-string', ~$nonEmptyString);
+	assertType('non-falsy-string', ~$nonFalsyString);
```

Comment:
This is not necessarily true - `~"\xcf"` results in `'0'` which is falsy

Reply:
fixed in 3061eb04229b747118eb7754517a2e34b2cd1afc - thanks